pydoclint/utils/return_yield_raise.py

```diff
@@ -114,8 +114,8 @@ def getRaisedExceptions(node: FuncOrAsyncFuncDef) -> list[str]:
 
 def _getRaisedExceptions(
         node: FuncOrAsyncFuncDef,
-) -> Generator[str, None, None]:
-    """Yield the raised exceptions in a function node"""
+) -> Iterator[str, None, None]:
```

Comment:
Hi @pechersky , using `Generator[str, None, None]` is preferred here.  You can read more here: https://jsh9.github.io/pydoclint/notes_generator_vs_iterator.html